### PR TITLE
BankAccount editing enabled

### PR DESCRIPTION
**Pull request by Titpost**

### CODE REVIEW

File: src/main/java/com/epam/lab/payments/web/html/DefaultController.java [hunk 1]
```diff
@@ -1,26 +1,68 @@
 package com.epam.lab.payments.web.html;
 
-import com.epam.lab.payments.web.html.cosmetic.CardNumberAdjuster;
+import com.epam.lab.payments.domain.UserEntity;
+import com.epam.lab.payments.dto.UserDTO;
```
Comment (iliaonishchenko) [inline src/main/java/com/epam/lab/payments/web/html/DefaultController.java]: Unused imports should be removed.

File: src/main/java/com/epam/lab/payments/web/AuthorizationController.java [hunk 2]
```diff
@@ -1,17 +1,20 @@
 package com.epam.lab.payments.web;
 
 import com.epam.lab.payments.UserValidator;
-import com.epam.lab.payments.domain.UserEntity;
+import com.epam.lab.payments.dto.UserDTO;
 import com.epam.lab.payments.services.AuthorizationService;
 import com.epam.lab.payments.services.SecurityService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.context.SecurityContextHolder;
```
Comment (iliaonishchenko) [inline src/main/java/com/epam/lab/payments/web/AuthorizationController.java]: Unused import should be removed.

File: src/main/java/com/epam/lab/payments/web/AuthorizationController.java [hunk 3]
```diff
@@ -25,14 +28,14 @@
     @RequestMapping(value = REGISTRATION, method = RequestMethod.GET)
     public ModelAndView registration(){
```
Comment (iliaonishchenko) [inline src/main/java/com/epam/lab/payments/web/AuthorizationController.java:26]: Private final field securityService never used.

**Review by iliaonishchenko — COMMENTED** [review]
Review done.

File: src/main/java/com/epam/lab/payments/web/AuthorizationController.java (hunk 2 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/web/AuthorizationController.java]: Code is outdated, now it is fine

File: src/main/java/com/epam/lab/payments/web/html/DefaultController.java (hunk 1 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/web/html/DefaultController.java]: Oh, yes, indeed, sorry

File: src/main/java/com/epam/lab/payments/web/AuthorizationController.java (hunk 3 above)
Comment (Titpost) [reply src/main/java/com/epam/lab/payments/web/AuthorizationController.java:26]: Really.
Done.
Thanks

**Review by iliaonishchenko — APPROVED** [review]
Review done.

**Pull request merged**